LeanKit-Labs/tag-release
# Defect dev rebase upstream

### Short description of the work completed

> Will update local master or develop when doing --dev before it creates the upstream feature-branch so you have the latest you are pushing up to create a PR against. Also, fixed quotes in changeReasons breaking tag-release during the --qa flow.

### Steps to test (if not obvious)

1. run --dev with an out-dated develop or master and have a local feature-branch that is up-to-date with upstream develop or master.
2. run --qa and then enter a change reason that has "double quotes" or 'single quotes' and it should strip them from the reason so it doesn't go 💥.

### For Reviewer Use Only

  patch)
